packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMainInput.tsx

```diff
@@ -20,7 +20,7 @@ function MaxButton(props: MaxButtonProps) {
   return (
     <button
       type="button"
-      className={`p-2 text-sm font-light text-gray-dark ${className}`}
+      className={`p-2 text-sm font-light text-gray-6 ${className}`}
```

Comment:
nit: Not introduced by the PR but we can go with twMerge here?

Reply:
6 and dark are different, this is the new colour on Figma